src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CallbackValidations.java

```diff
@@ -101,4 +106,58 @@ private CallbackValidations() {
     static Validation<String, List<Map<String, Object>>> hasAScannedRecord(CaseDetails theCase) {
         return scannedDocumentValidator.validate(theCase);
     }
+
+    @Nonnull
+    static Validation<String, Void> hasValidCombinationOfEventIdAndClassification(
+        CaseDetails theCase,
+        String eventId
+    ) {
+        return getJourneyClassification(theCase)
+            .map(classification -> {
+                    switch (classification) {
+                        case SUPPLEMENTARY_EVIDENCE:
+                            return ATTACH_TO_EXISTING_CASE.equalsIgnoreCase(eventId)
+                                ? Validation.<String, Void>valid(null)
```

Comment:
If we statically imported `valid` and `invalid`, we could perhaps be able to get rid of all occurrences of `Validation.<String, Void>`. To consider.

Reply:
I thought that but it is not allowing to the method return type `Validation<String, Void>` if we do static imports on valid and invalid. Discussed with @doncem  and this seems to be the only way as of now.

Reply:
got to find some love for this ladder 🤗